CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * Added support for create list of trackers
 * Added support for update brand for user
 * Removed `retrieveRates()` method because the shipment object already has rates. If you need to get new rates for a shipment, please use regenerateRates() method instead
+* Add `RetrieveMe()` Convenience Function that allow users to retrieve without specifying an ID.
```

Comment:
When referencing a function, you need to reference it correctly for the language you are working in.
```suggestion
* Add `retrieveMe()` Convenience Function that allow users to retrieve without specifying an ID.
```